tests/admin.test.ts

```diff
@@ -20,31 +20,31 @@ describe('retroactive attendance submission', () => {
     const conn = await DatabaseConnection.get();
     const users = UserFactory.create(3);
     const emails = users.map((user) => user.email);
-    const [proxyUser] = UserFactory.with({ accessType: UserAccessType.ADMIN });
+    const [adminUser] = UserFactory.with({ accessType: UserAccessType.ADMIN });
     const [event] = EventFactory.create(1);
```

Comment:
nit: this should remain as `proxyUser`. The reason we have it named `proxyUser` here, but not elsewhere, is because this user is submitting attendance on behalf of the other users, and is hence a "proxy" (see definition 3 of [proxy definition](https://www.merriam-webster.com/dictionary/proxy)). We originally had it named as `onBehalfOfUser` but it didn't flow off the tongue as easily

Reply:
I think `admin`'s fine since it's an admin account.

Reply:
I think `admin` would make the most sense, since the bonus route can only be used by an `admin` user anyway, so it also suggests proper flow for the endpoint as well.

Resolved in `df1a901`.